tests/features/gpu_step_definitions_test.go

```
@@ -0,0 +1,1230 @@
+	namespace := tc.reqHeaders["X-Tenant"]
+	if namespace == "" {
+		namespace = "test-tenant"
+	}
+
+	cmd := exec.Command("oc", "get", "pod", "-n", namespace, "-l",
+		fmt.Sprintf("job_id=%s", id),
+		"-o", "jsonpath={.items[0].status.phase}")
+	output, err := cmd.CombinedOutput()
+	if err != nil {
+		// Pod might not exist yet, which is fine
+		return nil
```

golangci-lint (2.12.2)

[error] error is not nil (line 884) but it returns nil

(nilerr)